odoo/sql_db.py

```diff
@@ -317,7 +315,9 @@ def execute(self, query, params=None, log_exceptions=True):
             if log_exceptions:
                 _logger.error("bad query: %s\nERROR: %s", tools.ustr(self._obj.query or query), e)
             raise
-        delay = real_time() - start
+        finally:
+            delay = real_time() - start
+            _logger.debug("[%.3f ms] query: %s", 1000 * delay, self._format(query, params))
```

Comment:
```suggestion
            if _logger.isEnabledFor(logging.DEBUG):
                _logger.debug("[%.3f ms] query: %s", 1000 * delay, self._format(query, params))
```

funny I've just created: https://github.com/odoo/odoo/pull/112371
Closing it in favor of this one

Reply:
It doesn't get it ? `_logger.debug` already does this condition before log the message.

Reply:
But not before `self._format` 😅

Reply:
In fact, it isn't only a tiny regression: it can takes +- 3 % of the time to the actual SQL query (empty database) ... my bad :grimacing: 

Nice catch @ivantodorovich  and thank you.